adventure/adventure.py

```diff
@@ -3068,71 +3087,72 @@ def E(t: str) -> str:
             god = await self.config.guild(self.bot.get_guild(guild_id)).god_name()
         msg = ""
         glyphs = False
-        glyphs_bonus = 0                       
+        glyphs_bonus = 0
+        pray_list_name = []
         for user in pray_list:
+            pray_list_name.append(self.E(user.display_name))
+        total_size = len(fight_list + talk_list + magic_list)
+        for user in magic_list:
             try:
                 c = await Character._from_json(self.config, user)
             except Exception:
                 log.error("Error with the new character sheet", exc_info=True)
                 continue
-            bonus_cleric = 0                   
-            if c.heroclass["name"] == "Cleric":
-                bonus_cleric = int((c.int + c.skill["int"] + c.att + c.skill["att"] + c.cha + c.skill["cha"])/3)
+            if c.heroclass["name"] == "Wizard" and not glyphs and len(pray_list) >= 1:

```

Comment:
Is it better to check "len(pray_list) >= 1" outside of the magic_list loop? 
Probably best to check before entering the loop... to save us iterating through all of the users if we don't have to

Reply:
Fair point

Reply:
done